app/models/classroom.rb

```diff
@@ -3,7 +3,8 @@
 # Table name: classrooms
 #
 #  id         :bigint           not null, primary key
-#  name       :string
+#  group      :string
```

Comment:
if we are changing like this, we also need to add grade (integer)
Math for first graders is different than math for fifth graders

Reply:
You got it!